# Motor feedback data as reported

std_msgs/Header header

float64 encoder_temp      # Measured encoder temperature in deg C
float64 motor_temp        # Measured internal motor temperature in deg C
float64 angular_speed     # Measured angular speed of the motor shaft (z axis) in rad/s
